_winners/saba-khukhunashvili.md

```diff
@@ -2,10 +2,10 @@
 name: Saba Khukhunashvili
 year: 2018
 org: JBoss Community
-country: GEO
+country: georgia
```

Comment:
Please, use Georgia flag emoji: 🇬🇪

Reply:
It's :georgia :, isn't it? :georgia:
Should I capitalize the first letter?

Reply:
o shout, i forgot : 🤣

Reply:
Browsers don't change those phrases into emojis, just paste there https://emojipedia.org/flag-for-georgia/

Reply:
Sorry for misunderstanding. It should work now.